src/main/java/com/academia/library/exception/handler/BookExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.academia.library.exception.handler;
+
+import com.academia.library.dto.ExceptionDto;
+import com.academia.library.exception.BookNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class BookExceptionHandler {
+
+    @ExceptionHandler(BookNotFoundException.class)
+    public ResponseEntity<ExceptionDto> handleBookNotFound(BookNotFoundException e) {
```

Comment:
And try to use something like this:
@ExceptionHandler(IllegalArgumentException.class)
public ResponseEntity<Object> handleIllegalArgument(IllegalArgumentException ex) {
log.error("Bad request: {} ", ex.getMessage());
return ResponseEntity.status(HttpStatus.BAD_REQUEST)
.body(ex.getMessage());
}

ExceptionDto is redundant

Reply:
Ok, thank